pyproject.toml

```diff
@@ -143,7 +143,7 @@ pytest-asyncio = "^0.21.1"
 [tool.poetry.extras]
 streamlit = ["streamlit"]
 community = ["llama-hub"]
-opensource = ["sentence-transformers", "torch", "gpt4all"]
+opensource = ["sentence-transformers", "torch", "gpt4all", "langchain"]
```

Comment:
I think langchain is already a dependency so no need to add it here separately.

Reply:
Wanted to add it in a shortcut so that devs can do pip install --upgrade embedchain[opensource]